specification/cognitiveservices/data-plane/ContentModerator/readme.md

```diff
@@ -4,20 +4,20 @@
 
 Configuration for generating Content Moderator SDK.
 
-The current release is `release_1_0`.
+The current release is `release_1_1`.
 
 ``` yaml
 
-tag: release_1_0
+tag: release_1_1
 ```
 # Releases
 
-### Release 1.0
-These settings apply only when `--tag=release_1_0` is specified on the command line.
+### Release 1.1
```

Comment:
Do not remove old releases!

Create a new release section and change the default, but don't remove the old one.